### DISCUSSION TITLE
I'm blocked from creating a new account

### DISCUSSION THREAD

**Discussion by plmts** [vercel/community#3413] [Help] [answered]
### Summary

I'm trying to create an account to make deploys of my personal project with vercel, but after i put my name and click on "continue", it shows this: 

"You have been blocked from creating an account. Request id: 79c30269-c73b-4972-8174-4b4b631a1c1b"

Don't tell me why i'm blocked.

And when i try to login with my git account, says that this account can't be found? So, anyone can help me?

### Steps to Reproduce

i dont now how. i don't know if its just with me or in general

**Comment by amyegan** [comment, marked as answer]
Hey @plmts. Sorry you're having trouble creating a new account. Since you will need to share sensitive account information, please email [registration@vercel.com](mailto:registration@vercel.com) so we can privately discuss these details. Be sure to include the email and username for the account in your message.

**Reply by plmts** [reply]
I already send an email to this adress with a video of my problem. But i will send it again, with the email and username that you requested.

**Reply by oezeb** [reply]
I too sent an email to that address on August 29(two days ago), got a case number and didn't here back from them yet. How long does it take to respond?

**Reply by leoaldamas** [reply]
I just had the same problem. I haven't even finished my sign up process, so I don't have an username, I guess.

**Reply by oezeb** [reply]
Follow up: I got a response from Vercel on the  4 of September(waited about a week) and the problem was solved.
